app/includes/header.php

```diff
@@ -0,0 +1,24 @@
+<header class="navbar">
+    <div class="container-fluid navbar-fixed-top bg-info">
+        <div class="navbar-brand text-primary">
+            BMcB
+        </div>
+        <a class="btn navbar-btn btn-primary" href="../index.php">
```

Comment:
../index.php takes you to the index outside of app. this should just be index.php (no dots or slash)
have you even tested this?!